src/EventListener/PostQueryBuilderSubscriber.php

```diff
@@ -115,21 +117,28 @@ protected function applyRequestFilters(QueryBuilder $queryBuilder, array $filter
     protected function applyFormFilters(QueryBuilder $queryBuilder, array $filters = array())
     {
         foreach ($filters as $field => $value) {
+
             $value = $this->filterEasyadminAutocompleteValue($value);
             // Empty string and numeric keys is considered as "not applied filter"
             if (null === $value || '' === $value || \is_int($field)) {
                 continue;
             }
+
             // Add root entity alias if none provided
             $field = false === \strpos($field, '.') ? $queryBuilder->getRootAlias().'.'.$field : $field;
+            $property = $field;
```

Comment:
I prefer `$ormField` instead of `$property`.

Reply:
fine by me